package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node index.js"
+    "start": "nohup node index.js &"
```

Comment:
Can we create a new command rather than changing the existing one?
Mㅁybe, `start:nohup` or something else?

Reply:
sure. The bash script will change accordingly